### PR TITLE
Fixes lavaland syndie base leading to space upon self destruct.

### LINKED ISSUE #33993: Lavaland syndicate base self-destruct literally breaches lavaland to space
[Round ID]: 81153

[Testmerges]: The following pull requests are currently test merged:
#33766: '[READY FOR TESTING] Refactors shuttles to be loaded in, rather than be on the station maps' by coiax at commit 49f8a0
#33979: 'Implements the Ballmer Peak: Now the bartender can help R&D!' by Iamgoofball at commit aeae5e

[Reproduction]:
1. activate lavaland self-destruct
2. wait for it to blow up
3. lavaland is exposed to space

wow

### LINKED ISSUE #34012: Lavaland syndicate base: Turret targeting
Related to: #33962
@WJohn 

**Issue**:
When enemies (fauna, miners, etc.) are at certain areas outside the lavaland syndie base, the turrets will shoot windows and depressurize rooms.
Look at this crappy MSPainted screenshot for an example:
https://i.imgur.com/Qsmw8NH.jpg

### PR DESCRIPTION
**Pull request by WJohn**
Fixes #33993 
Whoops!

EDIT:
Also now fixes #34012

### CODE REVIEW

**Comment by ShizCalev** [comment]
honk

**Comment by WJohn** [comment]
Kept telling myself "don't forget to put those in at the end" and then reasoning "yeah yeah I'll do it after I put in the areas" and then I totally fuckin' forgot.

**Comment by KorPhaeron** [comment]
Should just remove the turrets, they are way overkill when the expected enemy is a miner with a 3 tile range gun

They'll also lure fauna into the base by shooting at them

File: _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm [hunk 1]
```diff
@@ -4962,6 +4962,13 @@
 /obj/structure/table,
 /obj/item/storage/toolbox/syndicate,
 /obj/item/device/multitool,
+/obj/machinery/button/door{
+	id = "lavalandsyndi_telecomms";
+	name = "Telecomms Blast Door Control";
+	pixel_x = 26;
+	pixel_y = 0;
```
Comment (ShizCalev) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm]: dv

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by Repukan** [comment]
Why, exactly, are you adding turrets to the base that's mean to be defended by the people inside only?

Are you going to make a bridge next so they don't get burnt by lava?

**Comment by WJohn** [comment]
Have you not seen the design of the new base at all? There's an external walkway which is defended by turrets, this is to allow for some windows to look outside without having local wildlife breaking in, because the turrets will protect it. It also allows you to do some actual sniping which is fun.

**Comment by Repukan** [comment]
Someone forgot to post pictures in the original pr so no I didn't see it.

**Comment by WJohn** [comment]
But I DID post pictures??

**Comment by Repukan** [comment]
Tbh at the time I commented I had only seen the attached pic and not the Dropbox link.

Seems really hugboxy still. Padded corners and all, they shouldn't have automatic defenses

**Pull request merged**